cognite/neat/graph/extractors/_graph_capturing_sheet.py

```diff
@@ -59,7 +78,11 @@ def extract_triples_from_sheet(self, separator: str = ",", namespace: str | None
         """
         if self.filepath is None:
             raise ValueError("File path to the graph capturing sheet is not provided!")
-        return extract_graph_from_sheet(self.filepath, self.rules, separator, namespace)
+        graph_capturing_sheet = read_graph_excel_file_to_table_by_name(self.filepath)
```

Comment:
I am wandering whether we should have an abstract `read` or `read_source` (or some similar name) method that would read source `object` from which we extract triples?

Reply:
Lets keep the interface minimal for now until we see a strong need for it. You can always add methods to specific implementations.